Documentation/tutorial/intro_toc.md

```diff
@@ -64,34 +64,60 @@ Property Name | Type              | Description
 ------------- | ----------------- | ---------------------------
 *name*        | string            | Specifies the title of the *TOC Item*.
 *href*        | string            | Specifies the hyperlink of the *TOC Item*.
-*homepage*    | string            | Specifies the homepage of the *TOC Item*. It is useful when *href* is linking to a folder.
-*uid*         | string            | Specifies the `uid` of the referenced file. If the value is set, it overwrites the value of *href*.
-*homepageUid* | string            | Specifies the `uid` of the homepage. If the value is set, it overwrites the value of *homepage*.
-*items*       | *TOC Item Object* | Specifies the children *TOC Items*s of current *TOC Item*.
+*items*       | *TOC Item Object* | Specifies the children *TOC Item*s of current *TOC Item*.
+
+**Advanced**: These properties is useful when a TOC links to or reference to another TOC, or links to a uid.
```

Comment:
After read the documentation, I'm still a bit confused about the combination of the following properties. Such as can I write href, tocHref, topicHref and homepage at the same time?

Maybe a clear way is giving user a combination of how to use those properties in different situation in the following 3 ways link.

Such as:
### Link to a folder
Syntax1:
- name: title   // <title name>
- href: sampleFolder/  // <should be end with />
- homepage: https:***  // a link when click the folder node

Syntax2:
...

### Link to a local file
Syntax1:
***

Syntax2:
***